runtime/doc/various.txt

```diff
@@ -19,9 +19,9 @@ CTRL-L			Clear and redraw the screen.  The redraw may happen
 							*:redr* *:redraw*
 :redr[aw][!]		Redraw the screen right now.  When ! is included it is
 			cleared first.
-			Useful to update the screen halfway executing a script
-			or function.  Also when halfway a mapping and
-			'lazyredraw' is set.
+                        Useful to update the screen halfway through executing
+                        a script or function (or a mapping if 'lazyredraw'
+                        set).
```

Comment:
Sadly, we must preserve tabs in the help files. Otherwise we will have a never-ending formatting mess. Can you change this back to tabs and make sure the other changes are as well?